fineract-provider/src/main/resources/META-INF/spring/jdbc.properties

```diff
@@ -20,4 +20,9 @@
 DRIVERCLASS_NAME:org.drizzle.jdbc.DrizzleDriver
 PROTOCOL:jdbc
 SUB_PROTOCOL:mysql:thin
-PORT:3306
\ No newline at end of file
+PORT:3306
+
+fineract_tenants_driver:org.drizzle.jdbc.DrizzleDriver
+fineract_tenants_url:jdbc:mysql:thin://localhost:3306/fineract_tenants
+fineract_tenants_uid:root
+fineract_tenants_pwd:mysql
```

Comment:
FTR: Seeing this surprised me a little at first (especially given that there's no equivalent before; given that the (now deleted) `fineract-provider/src/test/resources/META-INF/context.xml` was scoped test only, but then I understood how this works. Given that the docker compose test passes proves that overriding these properties actually really is working, so all good.